### PR TITLE
fix(Heightmap): save z scale setting

### PR DESCRIPTION
**Pull request by vajonam**
- saves the z-max heightmapScale. 
- increased the scale size to 2mm. useful for follks with not very flat beds.

### CODE REVIEW

**Comment by vajonam** [comment]
Peek 2022-11-28 08-32

File: src/pages/Heightmap.vue [hunk 1]
```diff
@@ -671,20 +670,20 @@ export default class PageHeightmap extends Mixins(BaseMixin, ControlMixin) {
         this.$store.dispatch('gui/saveSetting', { name: 'view.heightmap.wireframe', value: newVal })
     }
 
-    get scale(): boolean {
-        return this.$store.state.gui.view.heightmap.scale ?? true
+    get scaleGradiant(): boolean {
```
Comment (th33xitus) [inline src/pages/Heightmap.vue:673]:
```suggestion
    get scaleGradient(): boolean {
```

File: src/pages/Heightmap.vue [hunk 2]
```diff
@@ -671,20 +670,20 @@ export default class PageHeightmap extends Mixins(BaseMixin, ControlMixin) {
...
+        return this.$store.state.gui.view.heightmap.scaleGradient ?? false
     }
 
-    set scale(newVal) {
-        this.$store.dispatch('gui/saveSetting', { name: 'view.heightmap.scale', value: newVal })
+    set scaleGradiant(newVal) {
```
Comment (th33xitus) [inline src/pages/Heightmap.vue:677]:
```suggestion
    set scaleGradient(newVal) {
```

File: src/pages/Heightmap.vue [hunk 3]
```diff
@@ -911,7 +910,7 @@ export default class PageHeightmap extends Mixins(BaseMixin, ControlMixin) {
     }
 
     get visualMapRange(): number[] {
-        if (!this.scaleVisualMap) return [-0.1, 0.1]
+        if (!this.scaleGradiant) return [-0.1, 0.1]
```
Comment (th33xitus) [inline src/pages/Heightmap.vue:913]:
```suggestion
        if (!this.scaleGradient) return [-0.1, 0.1]
```

File: src/pages/Heightmap.vue [hunk 4]
```diff
@@ -96,7 +96,7 @@
                                 <v-col
                                     class="col-12 col-sm-auto pt-0 pb-0 pl-lg-6 d-flex justify-center justify-sm-start">
                                     <v-switch
-                                        v-model="scaleVisualMap"
+                                        v-model="scaleGradiant"
```
Comment (th33xitus) [inline src/pages/Heightmap.vue:99]:
```suggestion
                                        v-model="scaleGradient"
```

**Review by th33xitus — CHANGES_REQUESTED** [review]
Concise spelling

**Review by th33xitus — APPROVED** [review]

**Pull request merged**